db/httpdb.js

```diff
@@ -439,6 +440,52 @@ module.exports = function (
     )
   }
 
+  DB.prototype.lockAccount = function (account) {
+    var unlockCode = crypto.randomBytes(16).toString('hex');
+    log.trace({ op: 'DB.lockAccount', uid: account && account.uid, unlockCode: unlockCode })
+
+    return this.pool.post(
+      '/account/' + account.uid.toString('hex') + '/lock',
+      {
+        lockedAt: Date.now(),
+        unlockCode: unlockCode
+      }
+    )
+  }
+
+  DB.prototype.unlockAccount = function (account) {
+    log.trace({ op: 'DB.unlockAccount', uid: account && account.uid })
+    return this.pool.post(
+      '/account/' + account.uid.toString('hex') + '/unlock'
+      )
+      .then(null,
+        function (err) {
+          if (err.statusCode === 404) {
+            err = error.unknownAccount()
```

Comment:
These two PRs have now been merged: https://github.com/mozilla/fxa-auth-db-server/pull/124 and https://github.com/mozilla/fxa-auth-db-mem/pull/17

So you shouldn't get a 404 from this operation any more (though of course, you may get some other error).